tests/ut/backends/test_redis.py

```diff
@@ -1,319 +1,268 @@
-from unittest.mock import ANY, AsyncMock, MagicMock, create_autospec, patch
+from unittest.mock import ANY, AsyncMock, MagicMock, patch
 
-import aioredis
 import pytest
+from redis.exceptions import ResponseError
 from tests.utils import Keys
 
-from aiocache.backends.redis import RedisBackend, RedisCache, conn
+from aiocache.backends.redis import RedisBackend, RedisCache
 from aiocache.base import BaseCache
 from aiocache.serializers import JsonSerializer
 
-pytest.skip("aioredis code is broken", allow_module_level=True)
-
-
-@pytest.fixture  # type: ignore[unreachable]
-def redis_connection():
-    return create_autospec(aioredis.RedisConnection)
-
 
 @pytest.fixture
-def redis_pool(redis_connection):
-    class FakePool:
-        def __await__(self):
-            yield
-            return redis_connection
+def redis_pipeline():
+    pipeline = AsyncMock(name="redis_pipeline")
```

Comment:
Can you try with `create_autospec()`? Code would be more robust, and the next few lines can probably be removed as well.

Reply:
This is also on my list of tasks (https://github.com/aio-libs/aiocache/issues/536#issuecomment-1068390586), so can be looked at separately if you've not got the time.